src/02-common-principles.md

```diff
@@ -152,6 +152,20 @@ correspond to a unique identifier of that subject, such as `01`.
 The same holds for the `session` entity with its `ses-` key and its `<label>`
 value.
 
+Session labels MUST consist only of alphanumeric characters `[a-zA-Z0-9]`
+and SHOULD be consistent across subjects.
+If numbers are used in session labels we RECOMMEND using zero padding
```

Comment:
This reads like a best practice, but RECOMMEND in RFC 2119 terms means "Do this unless you have a very good reason not to", and I don't think it's justified.

Reply:
With my proposal from https://github.com/bids-standard/bids-specification/pull/805#discussion_r644656450 this whole part would be pruned, because we already defined what a "label" constitutes above.